lib/osf-components/addon/components/registries/hierarchical-list/item/template.hbs

```diff
@@ -0,0 +1,30 @@
+<div data-test-item={{@manager.item.id}}>
+    {{input type='checkbox' change=(action @manager.onChange) checked=@manager.itemChecked disabled=@manager.isRoot}}
+    <OsfButton 
+        local-class='expand-button' 
+        @type='link' 
+        @onClick={{action @manager.toggleShowChildren}}
+        data-test-expand-child={{@manager.item.id}}
+    >
+        {{@manager.item.title}}
+        {{#if @manager.item.children}}
+            <FaIcon 
+                @icon='caret-{{if @manager.isShowChildren 'up' 'down'}}' 
+                @fixedWidth={{true}} 
+            />
+        {{/if}}
+    </OsfButton>
+</div>
+{{#if (and @manager.item.children @manager.isShowChildren)}}
+    <div local-class='children'>
```

Comment:
Hmm, this would most correctly be `<ul>` (and each item a `<li>`), do you think that'd be reasonable?